lib/livebook/hubs/team_client.ex

```diff
@@ -397,6 +406,34 @@ defmodule Livebook.Hubs.TeamClient do
     |> dispatch_deployment_groups(agent_connected)
   end
 
+  defp handle_event(:agent_key_created, agent_key_created, state) do
+    agent_key = build_agent_key(agent_key_created)
+
+    case find_deployment_group(agent_key.deployment_group_id, state.deployment_groups) do
+      nil ->
+        state
+
+      deployment_group ->
+        agent_keys = Enum.uniq([agent_key | deployment_group.agent_keys])
+        deployment_group = %{deployment_group | agent_keys: Enum.reverse(agent_keys)}
```

Comment:
This effectively reverses existing keys and then puts the new one at the end, probably not what we want?

Reply:
I could sort by ID in this case, should be a better choice here.

Reply:
Sorting by _something_ sounds like a good plan.